index.html

```diff
@@ -14,16 +14,31 @@ <h1 id="page-title">INTENTION TIMER</h1>
       <h2 class="activity-title" id="new-activity">New Activity</h2>
       <form id="user-activity-input">
         <label class="form-text" id="category">Select a category:<br />
-          <input class="activity-button" id="study" type="image" name="study" value="study" src="./assets/study.svg"/>
-          <input class="activity-button" id="meditate" type="image" name="meditate" value="meditate" src="./assets/meditate.svg"/>
-          <input class="activity-button" id="exercise" type="image" name="exercise" value="exercise" src="./assets/exercise.svg"/>
+          <div class="activity-card">
```

Comment:
Let's make sure to go back through our CSS and line up our new class/id values with the newly organized HTML elements